Update Firefox Android cert instruction (#909)
AdguardTeam/AdguardKnowledgeBase: 04.android/06.solving-problems/13.firefox-certificates/docs.en.md

#### Method 1

> > This method works in Firefox Android versions 90 and later. 

Reviewer: ```suggestion
> This method works in Firefox for Android version 90.0 and later. 
```

> To make Firefox trust AdGuard certificate, follow this instruction:

Reviewer: ```suggestion
To make Firefox trust the AdGuard certificate, do the following:
```

1. Run the browser.
2. Go to *Settings > About Firefox*.

<img src="https://cdn.adguard.com/public/Adguard/kb/Firefox_cert/ff_nightly_about_en.jpg" style="border: 1px solid #efefef; max-width: 350px; padding: 2px;">

> 3. Rapidly tap several times on the Firefox logo at the top of the screen.

Reviewer: ```suggestion
3. Tap the Firefox logo five times.
```
